lib/app/views/languages/_getting_started.erb

```diff
@@ -2,5 +2,6 @@
   <div class="article-contents">
     <h2 class="header">Getting Started</h2>
     <p>Need help getting started? You can find install documentation and resources for <%= language %> <a href="http://help.exercism.io/getting-started-with-<%= slug %>.html">here</a>.</p>
+    <p>Also here is the repository with the all the problems <a href=<%= repo_url %>>repository</a>.</p>
```

Comment:
Let's put this in a separate section.

The header might be **Having Issues?**, and then the text could be:

_If you find typos or inconsistencies in any of the problems, or if anything is unclear, post a new issue in the repository for this track._